modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowUsecase.testcase

```diff
@@ -796,6 +796,114 @@ definition {
 			wikiPageTitle = "Wiki Page Title");
 	}
 
+	@description = "LPS-193788: Verify if it is not possible to publish a workflow at a virtual instance when the configuration is disabled at main instance"
+	@priority = 3
+	test CanNotPublishWorkflowAtVirtualInstance {
```

Comment:
@Luizcarlosqueiroz ohh I guess we can just rename to CanNotAddWorkflowProcessInAVirtualInstance to make it clear that the plus button will not be there. We should also include in the description that it is the default behavior.